conventions/src/main/kotlin/otel.animalsniffer-conventions.gradle.kts

```diff
@@ -10,4 +12,8 @@ dependencies {
 
 animalsniffer {
   sourceSets = listOf(java.sourceSets.main.get())
+}
+tasks.withType<AnimalSniffer> {
```

Comment:
`tasks.withType<Foo>.configureEach {`

Reply:
What's the difference to the current way?